### ISSUE TITLE
"...is not a directory!!" multiple errors

### ISSUE THREAD

**Issue by avttrue** [QuasarApp/CQtDeployer#809]
**OS:** Linux 6.5.0-35-generic #35~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Tue May  7 09:00:52 UTC 2 x86_64 x86_64 x86_64 GNU/Linux (KDE Neon, wayland)

**CQtDeployer version :** 1.6.2351.ec4840f

**command line:** cqtdeployer -bin MY_APP -extraLibs "libb2,libxcb-cursor,libvdpau_va_gl" -enablePlugins "wayland-graphics-integration-client,wayland-graphics-integration-server" -disablePlugins "platformthemes,styles,wayland-decoration-client" -verbose 3 >&verboselog.txt

I get a lot of errors like "...is not a directory!!". Even the "DistributionKit" catalog is not created.

[verboselog.txt](https://github.com/QuasarApp/CQtDeployer/files/15392167/verboselog.txt)

**Comment by EndrII** [comment]
Try to add -qmake ... option to your deployment command

**Comment by avttrue** [comment]
> Try to add -qmake ... option to your deployment command

But I not use qmake. This is cmake project.

**Comment by EndrII** [comment]
But it looks as you tried to deploy app in forbidden for snap directory, make sure that you run cqtdeployer in /home or /media dirs.

**Comment by avttrue** [comment]
No...
`~/Projects/Projects_QT/build-karass-Desktop_Qt_6_7_0-Release$ cqtdeployer -bin karass -extraLibs "libb2,libxcb-cursor,libvdpau_va_gl" -enablePlugins "wayland-graphics-integration-client,wayland-graphics-integration-server" -disablePlugins "platformthemes,styles,wayland-decoration-client" -verbose 3 >&verboselog.txt`

**Comment by EndrII** [comment]
```
Error: The deployment target requires Qt libraries, but initialize of Qt directories is failed. Please use the qmake option to set a path to the qmake executable.
```
Do you're sure that your deployed binaries not require qt ? Because cqtdeployer detect qt dependencies. 
if you want to ignore this error, just use the 
**noQt** option

**Comment by EndrII** [comment]
this option, will ignore this message, and desable deployement of the 

> > Try to add -qmake ... option to your deployment command
> 
> But I not use qmake. This is cmake project.

yse but the -qmake option just sets qt pathes for cqtdeployer, you must use this option if you deploy qt project 
if you deploy not qt projet you must use the noQt option.

**Comment by EndrII** [comment]
see this issue https://github.com/QuasarApp/CQtDeployer/issues/788

**Comment by avttrue** [comment]
> qmake

Yes, the `qmake` option helped (it seems) the distribution was assembled, but the errors remained. I just don't know how successful the build was.

[verboselog.txt](https://github.com/QuasarApp/CQtDeployer/files/15393335/verboselog.txt)

**Comment by avttrue** [comment]
> 
> Do you're sure that your deployed binaries not require qt ? Because cqtdeployer detect qt dependencies. if you want to ignore this error, just use the **noQt** option

```
Error: The deployment target requires Qt libraries, but initialize of Qt directories is failed. Please use the qmake option to set a path to the qmake executable.
```

The fact is that I previously built distribution kit using this command. But I recently reinstalled Qt.

**Comment by EndrII** [comment]
> > qmake
> 
> Yes, the `qmake` option helped (it seems) the distribution was assembled, but the errors remained. I just don't know how successful the build was.
> 
> [verboselog.txt](https://github.com/QuasarApp/CQtDeployer/files/15393335/verboselog.txt)

About what error you are talking ?

**Comment by avttrue** [comment]
> About what error you are talking ? output log looks as fine

About `is not a directory!!` - There were no such messages before.

**Comment by avttrue** [comment]
> see this issue #788

`-qtRootDir` option would be more understandable, by the way. There is not yet?

**Comment by avttrue** [comment]
I think the problem is solved, thank you!

**Comment by EndrII** [comment]
okay close please this issue, if all works fine
